src/test/java/user/TC_UserUpload.java

```diff
@@ -0,0 +1,99 @@
+package user;
+
+import api.training.dto.Sex;
+import api.training.dto.UserDto;
+import api.training.services.UserService;
+import api.training.services.ZipCodeService;
+import com.beust.jcommander.internal.Lists;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import org.testng.internal.collections.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TC_UserUpload {
```

Comment:
Add test, when file is invalid (incorrect json structure f.e.)

Reply:
Added.